CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
   making Gleam packages discoverable through global search of HexDocs.
   ([Diemo Gebhardt](https://github.com/diemogebhardt))
 
+- Respect the `GLEAM_CACERTS_PATH` environment variable
```

Comment:
Could you explain what this new feature is please 🙏 It's not clear what respect means, and it also implies there was something it was supposed to do previously that it did not